qiita_db/test/test_study.py

```diff
@@ -179,6 +179,54 @@ def _change_processed_data_status(self, new_status):
             "UPDATE qiita.processed_data SET processed_data_status_id = %s",
             (id_status,))
 
+    def test_get_info(self):
```

Comment:
Something on qiita_db cannot be safely tested through something on qiita_pet... please add test here.